packages/@sfx/sayt-plugin/src/index.ts

```diff
@@ -3,14 +3,18 @@
  */
 const testObj = {
   label: 'test text',
+  number: 6,
 }
 
-const testFunc = (text: TestInterface) => {
-  console.log(`I am logging out ${ text.label }`);
+export const testFunc = (data: DummyInterface) => {
+  return exports.getLabel(data).toUpperCase();
```

Comment:
I'm not sure we should be doing this, but we're not going to be keeping this, so I'll ignore it for now.